geomstats/numerics/geodesic.py

```diff
@@ -530,7 +490,11 @@ def _create_grid(self):
         return gs.linspace(0.0, 1.0, num=self.n_nodes)
 
     def _default_initialization(self, space, point, base_point):
-        point_0, point_1 = base_point, point
+        if point.ndim == 1:
```

Comment:
Docstring

Reply:
(not necessarily to explain what the function does, that's explicit from the code, but *why* we need that)

Reply:
Can you be more explicit on the part that is not clear?

This relates with the algorithm we use for solving this problem. Since we solve the ODE as a boundary value problem, we need to provide an initialization for the values at each node.